R/render-cached-plot.R

```diff
@@ -150,6 +150,7 @@
 #'   `"app"` (the default), `"session"`, or a cache object like
 #'   a [diskCache()]. See the Cache Scoping section for more
 #'   information.
+#' @param alt Optional parameter to pass `alt` to the img-tag.
```

Comment:
I believe this can be removed since we have the `@inheritParams renderPlot` above

```suggestion
```